ddtrace/contrib/celery/signals.py

```diff
@@ -156,6 +160,11 @@ def trace_after_publish(*args, **kwargs):
     if span is None:
         return
     else:
+        nodename = span.get_tag("celery.hostname")
+        if nodename is not None:
+            _, host = nodenames.nodesplit(nodename)
+            span.set_tag_str(net.TARGET_HOST, host)
```

Comment:
I gave you the typo'd version of my diff 🤦 
```suggestion
            span.set_tag_str(net.TARGET_HOST, nodename)
```